openhands/runtime/utils/runtime_build.py

```diff
@@ -197,6 +200,60 @@ def build_runtime_image(
     docker_build_folder: str | None = None,
     dry_run: bool = False,
     force_rebuild: bool = False,
+) -> str:
```

Comment:
Are you able to add method docs?
Would be really nice if we could get into commenting more often so others can understand easier.

Reply:
I will as I mentioned in the PR post, once it's ready. :)

Reply:
yeah - no only docstrings - i think it will be good to also copy over the inline comments.